ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/assertion/ExpertReportRowAssert.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.cmc.ccd.assertion;
+
+import org.assertj.core.api.AbstractAssert;
+import uk.gov.hmcts.cmc.ccd.domain.directionsquestionnaire.CCDExpertReportRow;
+import uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.ExpertReportRow;
+
+import java.util.Objects;
+
+public class ExpertReportRowAssert extends AbstractAssert<ExpertReportRowAssert, ExpertReportRow> {
+
+    public ExpertReportRowAssert(ExpertReportRow actual) {
+        super(actual, ExpertReportRowAssert.class);
+    }
+
+    public ExpertReportRowAssert isEqualTo(CCDExpertReportRow ccdExpertReportRow) {
```

Comment:
just a question, are we explicitly creating assertions for these objects because we want to ignore the `id` when asserting for equality in tests? Because if so, we can avoid having this assertion and you can just make sure you have the same id in tests :) it would avoid extra useless classes.

Reply:
I would like to know answer to that question too!! Again, i just followed existing code style. I would have preferred to understand a bit more and work on this story (for my learning) but under pressure to get it done :(

Reply:
Forget what I asked - I didn't read properly that we are equalling two different things (`CCDExpertReportRow` and `ExpertReportRow` 😬 ) so I can understand the assertion. Soz. Carry on.